src/pages/membership/components/approveYouthProfileForm/ApproveYouthProfileForm.tsx

```diff
@@ -193,30 +192,37 @@ function ApproveYouthProfileForm(props: Props) {
                 name="approverPhone"
                 type="tel"
                 invalid={props.submitCount && props.errors.phone}
-                invalidText={
-                  props.submitCount &&
-                  props.errors.phone &&
-                  t(props.errors.phone)
+                helperText={
+                  props.submitCount && props.errors.phone
+                    ? t(props.errors.phone)
+                    : ''
                 }
                 labelText={t('approval.phone')}
               />
             </div>
             <ul className={styles.terms}>
-              <Field name="terms" type="checkbox" />
-              <span className={styles.listLabel}>
-                {t('approval.approveTermsText_1')}
-                <Link
-                  to="/terms-of-service"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {t('approval.approveTermsText_link')}
-                </Link>
-                {t('approval.approveTermsText_2')}
-              </span>
+              <Field
+                name="terms"
+                type="checkbox"
+                as={Checkbox}
+                labelText={
+                  <span className={styles.listLabel}>
+                    {t('approval.approveTermsText_1')}{' '}
+                    <Link
+                      to="/terms-of-service"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      {t('approval.approveTermsText_link')}
+                    </Link>
+                    {t('approval.approveTermsText_2')}
+                  </span>
```

Comment:
I'll change this Trans component later. Few other translations have been changed in excel and updating them now will probably cause problems somewhere in the application.